apps/vmq_commons/src/vmq_types_mqtt5.hrl

```diff
@@ -48,6 +48,7 @@
 -define(M5_MAX_CONNECT_TIME, 16#A0).
 -define(M5_SUBSCRIPTION_IDS_NOT_SUPPORTED, 16#A1).
 -define(M5_WILDCARD_SUBS_NOT_SUPPORTED, 16#A2).
+-define(M5_TCP_CLOSED, 16#A3).
```

Comment:
What is 16#A3?

Reply:
Let's not do any kind of protocol change for this feature.

Reply:
Removed